elastic/kibana
# [Security Solution] updates host risk score decimal count (UI)

rounds the long number on presentation 

_before_
<img width="650" alt="Screen Shot 2021-11-18 at 11 46 59 AM" src="https://user-images.githubusercontent.com/18617331/142680504-bff09dbc-31fa-461d-bff0-93936de7d088.png">

_after_
<img width="618" alt="Screen Shot 2021-11-19 at 3 17 37 PM" src="https://user-images.githubusercontent.com/18617331/142686359-8af080d3-dfe7-470d-8e11-045a69110e31.png">


## testing
- follow the instructions in [host-risk-score.md](https://github.com/elastic/detection-rules/blob/main/docs/experimental-machine-learning/host-risk-score.md) to set up the host risk score functionality. 
- Add `xpack.securitySolution.enableExperimental: ['riskyHostsEnabled']` to kibana.dev.yml.
- create an alert with a host.name value
- confirm that the host risk score card UI displays the score with two decimals
